Port softplus_backward to structured

Freey0 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #58661 Port silu_backward to structured
* #58665 Port gelu_backward to structured
* #58664 Port gelu to structured
* #58663 Port hardshrink to structured
* #58662 Port hardshrink_backward and softshrink_backward to structured
* #58660 Port elu_backward to structured
* #58484 Port hardsigmoid_backward to strucutred
* #58483 Port leaky_relu_backward to structured
* **#58482 Port softplus_backward to structured**



Differential Revision: [D28571059](https://our.internmc.facebook.com/intern/diff/D28571059)

Freey0 committed:
Port softplus to structured

[ghstack-poisoned]

ezyang commented:
@ezyang has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28571059).

Freey0 committed:
Update on "Port softplus to structured"


Differential Revision: [D28571059](https://our.internmc.facebook.com/intern/diff/D28571059)

[ghstack-poisoned]

ezyang commented:
@ezyang has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28571059).

Freey0 committed:
Update on "Port softplus-backward to structured"




Differential Revision: [D28571059](https://our.internmc.facebook.com/intern/diff/D28571059)

[ghstack-poisoned]

Freey0 committed:
Update on "Port softplus_backward to structured"




Differential Revision: [D28571059](https://our.internmc.facebook.com/intern/diff/D28571059)

[ghstack-poisoned]

ezyang commented:
@ezyang has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D28571059).
